release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -3066,3 +3066,19 @@ link:https://access.redhat.com/solutions/6967084[{product-title} 4.10.22 contain
 ==== Updating
 
 To update an existing {product-title} 4.10 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster within a minor version by using the CLI] for instructions.
+
+[id="ocp-4-10-23"]
+=== RHBA-2022:5568 - {product-title} 4.10.23 bug fix update
+
+Issued: 2022-07-20
+
+{product-title} release 4.10.23 is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHBA-2022:5568[RHBA-2022:5568] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2022:5567[RHBA-2022:5567] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+link:https://access.redhat.com/solutions/6968082[{product-title} 4.10.23 container image list]
+
+[id="ocp-4-10-23-bug-fixes"]
+==== Bug fixes
+
+* Previously the Ingress Operator would detect changes made through the Ingress Controller and set the Ingress Operator’s “Upgradeable” status condition to “False” blocking upgrades. With this update, the Ingress Operator no longer blocks upgrades. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2097735[*BZ#2097735*])
```

Comment:
You could change "the Ingress Operator’s “Upgradeable” status condition" to the following, to avoid using a possessive:

>  the “Upgradeable” status condition of the Ingress Operator

Reply:
You could split the first sentence into two, to facilitate localisation and enhance readability. For example:

> Previously, the Ingress Operator detected changes made through the Ingress Controller and set the `Upgradeable` status condition of the Ingress Operator to `False`. The `False` status condition blocked upgrades.

In the example, I have also applied back-ticks to the status name and value. My assumption is that they refer to code elements, but please correct me if I am wrong.